packages/auth/src/api/account_management/mfa.test.ts

```diff
@@ -159,6 +161,139 @@ describe('api/account_management/finalizeEnrollPhoneMfa', () => {
   });
 });
 
+describe('api/account_management/startEnrollTotpMfa', () => {
+  const request = {
+    idToken: 'id-token',
+    totpEnrollmentInfo: {}
+  };
+
+  let auth: TestAuth;
+
+  beforeEach(async () => {
+    auth = await testAuth();
+    mockFetch.setUp();
+  });
+
+  afterEach(mockFetch.tearDown);
+
+  it('should POST to the correct endpoint', async () => {
+    const currentTime = new Date().toISOString();
+    const mock = mockEndpoint(Endpoint.START_MFA_ENROLLMENT, {
+      totpSessionInfo: {
+        sharedSecretKey: 'key123',
+        verificationCodeLength: 6,
+        hashingAlgorithm: 'SHA256',
+        periodSec: 30,
+        sessionInfo: 'session-info',
+        finalizeEnrollmentTime: currentTime
+      }
+    });
+
+    const response = await startEnrollTotpMfa(auth, request);
+    expect(response.totpSessionInfo.sharedSecretKey).to.eq('key123');
+    expect(response.totpSessionInfo.verificationCodeLength).to.eq(6);
+    expect(response.totpSessionInfo.hashingAlgorithm).to.eq('SHA256');
+    expect(response.totpSessionInfo.periodSec).to.eq(30);
+    expect(response.totpSessionInfo.sessionInfo).to.eq('session-info');
+    expect(response.totpSessionInfo.finalizeEnrollmentTime).to.eq(currentTime);
+    expect(mock.calls[0].request).to.eql(request);
+    expect(mock.calls[0].method).to.eq('POST');
+    expect(mock.calls[0].headers!.get(HttpHeader.CONTENT_TYPE)).to.eq(
+      'application/json'
+    );
+    expect(mock.calls[0].headers!.get(HttpHeader.X_CLIENT_VERSION)).to.eq(
+      'testSDK/0.0.0'
+    );
+  });
+
+  it('should handle errors', async () => {
+    const mock = mockEndpoint(
+      Endpoint.START_MFA_ENROLLMENT,
+      {
+        error: {
+          code: 400,
+          message: ServerError.INVALID_ID_TOKEN,
+          errors: [
+            {
+              message: ServerError.INVALID_ID_TOKEN
+            }
+          ]
+        }
+      },
+      400
+    );
+
+    await expect(startEnrollTotpMfa(auth, request)).to.be.rejectedWith(
+      FirebaseError,
+      "Firebase: This user's credential isn't valid for this project. This can happen if the user's token has been tampered with, or if the user isn't for the project associated with this API key. (auth/invalid-user-token)."
```

Comment:
Nit -- just check for the actual error code instead of the full error message (in some of our tests we do the whole message but it's brittle and I've been trying to move us away from that)

You can just do `to.be.rejectedWith(FirebaseError, 'auth/invalid-user-token')`

Reply:
Done.